code/modules/SCP/SCPs/SCP-1507.dm

```diff
@@ -0,0 +1,68 @@
+/mob/living/simple_animal/hostile/retaliate/cooperative/scp1507
+	name = "pink flamingo"
+	desc = "A ping flamingo that acts like a real one."
+	icon = 'icons/SCP/scp-1507.dmi'
+	maxHealth = 100
+	health = 100
+	icon_state = "flamingo"
+	icon_living = "flamingo"
+	icon_dead = "dead"
+	var/icon_resting = "sleep"
+	response_help = "strokes"
+	response_disarm = "gently pushes aside"
+	response_harm = "hits the"
+
+	harm_intent_damage = 3
+	var/enrage_potency = 4
+	var/max_damage = 8
+
+	ai_holder_type = /datum/ai_holder/simple_animal/retaliate/cooperative/scp1507
+	say_list_type = /datum/say_list/scp1507
+	natural_weapon = /obj/item/natural_weapon/scp1507
+
+/mob/living/simple_animal/hostile/retaliate/cooperative/scp1507/Initialize()
+	. = ..()
+	SCP = new /datum/scp(
+		src, // Ref to actual SCP atom
+		"pink flamingo", //Name (Should not be the scp desg, more like what it can be described as to viewers)
+		SCP_EUCLID, //Obj Class
+		"1507" //Numerical Designation
```

Comment:
Worth mentioning, but we might want to have to count the amount of flamingos, and name them appropriately, i.e. SCP 1507-1, SCP 1507-2 and so on.